linuxdeploy-plugin-gtk.sh

```diff
@@ -48,10 +58,69 @@ fi
 
 mkdir -p "$APPDIR"
 
-pushd "$APPDIR"
+echo "Installing AppRun hook"
+HOOKSDIR="$APPDIR/apprun-hooks"
+HOOKFILE="$HOOKSDIR/linuxdeploy-plugin-gtk.sh"
+mkdir -p "$HOOKSDIR"
+cat > "$HOOKFILE" <<\EOF
+#! /bin/bash"
+
+gsettings get org.gnome.desktop.interface gtk-theme 2> /dev/null | grep -qi "dark" && GTK_THEME_VARIANT="dark" || GTK_THEME_VARIANT="light"
+APPIMAGE_GTK_THEME=${APPIMAGE_GTK_THEME:-"Adwaita:$GTK_THEME_VARIANT"} # Allow user to override theme (discouraged)
```

Comment:
Looks a bit like you assume people have Adwaita installed. Is that a safe assumption?

By the way, should be `APPIMAGE_GTK_THEME="${APPIMAGE_GTK_THEME:-"Adwaita:$GTK_THEME_VARIANT"}"` (looks weird, I know, had to learn this, too... maybe run `shellcheck` on the script).

Reply:
> Looks a bit like you assume people have Adwaita installed. Is that a safe assumption?

Yes, Adwaita is built-in since GTK3 (see `/usr/share/themes/Adwaita/gtk-3.0/gtk.css`):
```css
/* Adwaita is now part of GTK+ 3, this file is no longer used */
```

> maybe run `shellcheck` on the script

Fixed, no warning at all.